classes/ActionScheduler_ListTable.php

```diff
@@ -444,6 +444,10 @@ protected function get_schedule_display_string( ActionScheduler_Schedule $schedu
 
 		$schedule_display_string = '';
 
+		if ( is_a( $schedule, 'ActionScheduler_NullSchedule' ) ) {
+			return 'async';
```

Comment:
@crstauf This needs to be translatable. Also, I've created a `version/3.2.0` branch that we will be merging to. If you could base future PRs against it that would be great :)

Reply:
> needs to be translatable

Made translatable in 590a793.

Reply:
> This needs to be translatable.

@rrennick Made translatable in 590a793.